bigquery_etl/cli/query.py

```diff
@@ -141,109 +176,131 @@ def create(name, sql_dir, project_id, owner, init):
     view_path = None
     path = Path(sql_dir)
 
-    if dataset.endswith("_derived"):
-        # create a directory for the corresponding view
-        derived_path = path / project_id / dataset / (name + version)
-        derived_path.mkdir(parents=True)
-
-        view_path = path / project_id / dataset.replace("_derived", "") / name
-        view_path.mkdir(parents=True)
+    if not no_schedule and not dag and not use_default_dag:
+        click.echo(
+            "Please specify the scheduling information or"
+            " skip using --no-schedule. "
+            "Run `bqetl query create --help` for more options.",
+            err=True,
+        )
```

Comment:
nit: if this is meant to exit, ignore if not. Removes the need for the `else` below
```suggestion
        )
        return
```